src/segment-loader.js

```diff
@@ -1109,9 +1125,12 @@ export default class SegmentLoader extends videojs.EventTarget {
       return;
     }
 
-    if (segmentInfo.timestampOffset !== null &&
-        segmentInfo.timestampOffset !== this.sourceUpdater_.timestampOffset()) {
-      this.sourceUpdater_.timestampOffset(segmentInfo.timestampOffset);
+    const timelineMapping = this.syncController_.mappingForTimeline(segmentInfo.timeline);
```

Comment:
der syncController hat keine Funktion mappingForTimeline, vielleicht hast du vergessen diese Funktion mit zu commiten ?

Reply:
you are right :+1: